### ISSUE TITLE
Discuss checking in static files.

### ISSUE THREAD

**Issue by meandavejustice** [mozilla/idea-town-app#17]
Would probably simplify build and deployments if we just checked in a `dist` directory. That way when someone is working on the server or deploying they don't need to mess with gulp and all of the dev dependencies.

**Comment by meandavejustice** [comment]
cc/ @mostlygeek Assuming this would greatly simplify deployments.

**Comment by mostlygeek** [comment]
Never check in what can be/is generated. 

We're moving towards docker container deployment so the Dockerfile should generate the assets as part of the build.

**Comment by lmorchard** [comment]
Yeah, I think a way to simplify this in dev (and prod) is to generate a Docker image with the built assets and a static web server (e.g. nginx). Grab the image from Docker Hub or somesuch. Could graduate to using a CDN in production with the static web server as origin.

**Comment by lmorchard** [comment]
I guess something else to consider is how to build assets for prod vs dev. That is, in dev we like file watchers that constantly re-build. In prod, we want to build once before deployment in the Dockerfile. Is there a fairly standard way of doing both? If so, let's do whatever everyone else does
